apps/modernization-ui/src/apps/search/investigation/CriteriaSearchFields.spec.tsx

```diff
@@ -0,0 +1,76 @@
+import { render, waitFor, screen } from '@testing-library/react';
+import { useForm } from 'react-hook-form';
+import { InvestigationFilterEntry } from './InvestigationFormTypes';
+import userEvent from '@testing-library/user-event';
+import CriteriaSearchFields from './CriteriaSearchFields';
+
+const InvestigationFormWithFields = () => {
+    const defaultSelectable = { name: '', value: '', label: '' };
+    const defaultValues: InvestigationFilterEntry = {
+        createdBy: defaultSelectable,
+        updatedBy: defaultSelectable,
+        investigator: defaultSelectable,
+        pregnancyStatus: defaultSelectable,
+        investigationStatus: defaultSelectable,
+        jurisdictions: [defaultSelectable],
+        conditions: [defaultSelectable],
+        caseStatuses: [defaultSelectable],
+        notificationStatuses: [defaultSelectable],
+        outbreaks: [defaultSelectable],
+        processingStatuses: [defaultSelectable],
+        programAreas: [],
+        reportingFacility: defaultSelectable
+    };
+
+    const investigationForm = useForm<InvestigationFilterEntry>({
+        defaultValues
+    });
+
+    return <CriteriaSearchFields form={investigationForm} />;
+};
+
+describe('CriteriaSearchFields', () => {
+    describe('Investigation Status', () => {
+        it('should contain default selection', async () => {
+            const { container } = render(<InvestigationFormWithFields />);
+
+            await waitFor(() => {
+                const preg = screen.getByTestId('investigationStatus');
```

Comment:
suggestion (non-blocking): The `getByTestId` should be used as a last resort [getByRole](https://testing-library.com/docs/queries/byrole) can be used here.  `getByRole('option', { name: 'the label text of the option' });`